### PR TITLE
feat: add meassing proof of proposition 5 of Division euclidienn

### PR DESCRIPTION
**Pull request by GabinDDL**
# Add the missing proofs of proposition 5 of the "Division euclidienne" section.

## Description

Add the missing proofs from proposition 5, i.e. implication 2) => 1) and 1) => 3).

## Changes Made

Add proofs in proposition 5 of "Division euclidienn"

## Checklist

- [x] I have self-reviewed my code
- [x] Code follows project's style guidelines
- [x] Tests added and passing
- [x] Documentation updated (if needed)

## @Mention 

@Yag000

### CODE REVIEW

File: algebre-II-notes.tex [hunk 1]
```diff
@@ -158,8 +158,24 @@ \subsection{Division euclidienne}
 
 	\[ ab \in <p>\quad  et \quad a \notin <b> \implies b \in <p> \implies <p> \text{ premier} \]
 
-	Le reste de la preuve est laissé en exercice (je n'arrive pas a bien lire ce qui est écrit dans le cours).
+\end{proof}
+
+\begin{proof}
+
+    2) $\implies$ 1):\\
+    Soit $\mathrm{P}$ un système de représentant des indestructibles.
```
Comment (Yag000) [inline algebre-II-notes.tex]:
```suggestion
    Soit $\mathcal{P}$ un système de représentant des irréductibles.
```

File: algebre-II-notes.tex [hunk 2]
```diff
@@ -158,8 +158,24 @@ \subsection{Division euclidienne}
...
+    On a alors $u \cdot \prod_{p \in \mathrm{P}} p^{n_q}$ (divisible par $q^{n_q} \in <q>$ ) $= v \cdot \prod_{p \in \mathrm{P}} p^{m_q}$ divisible par $q^{m_q} \in <q>$)
```
Comment (Yag000) [inline algebre-II-notes.tex]:
```suggestion
	\begin{equation*}
		u \cdot \prod_{p \in \mathcal{P}} p^{n_q} (\text{divisible par } q^{n_q} \in <q> ) = v \cdot \prod_{p \in \mathcal{P}} p^{m_q} \text{divisible par } q^{m_q} \in <q>)
	\end{equation*}
	\noindent
```

File: algebre-II-notes.tex [hunk 3]
```diff
@@ -158,8 +158,24 @@ \subsection{Division euclidienne}
...
+    Si il existe $q \in \mathbb{P}$ tel que $m_q > n_q$, alors $q$ divise $u \cdot \prod_{p \ne q} p^{n_q}$ ($\in <q>$), ce qui n'est pas possible par 2).
```
Comment (Yag000) [inline algebre-II-notes.tex]:
```suggestion
    Si il existe $q \in \mathcal{P}$ tel que $m_q > n_q$, alors $q$ divise $u \cdot \prod_{p \ne q} p^{n_q}$ ($\in <q>$), ce qui n'est pas possible par 2).
```

File: algebre-II-notes.tex [hunk 4]
```diff
@@ -158,8 +158,24 @@ \subsection{Division euclidienne}
...
+
+\end{proof}
+
+\begin{proof}
 
+    1) $\implies$ 3):\\
+    $A$ est factoriel. Si $a$  divise $x$, on écrit $a$, $b$, $c$ sous la forme $u \cdot \prod_{p \in \mathrm{P}} p^{v_{p(x)}}$.
```
Comment (Yag000) [inline algebre-II-notes.tex]:
```suggestion
    $A$ est factoriel. Si $a$  divise $x$, on écrit $a$, $b$, $c$ sous la forme $u \cdot \prod_{p \in \mathcal{P}} p^{v_{p(x)}}$.
```

Comment (Yag000) [reply algebre-II-notes.tex]: Je suppose que $\mathcal{P}$ est un système de répresentants des irrédutibles de $A$? Si cest le ca il faut l'indiquer, car cela a été définit dans une autre démo, pas dans celle là.

File: algebre-II-notes.tex [hunk 5]
```diff
@@ -158,8 +158,24 @@ \subsection{Division euclidienne}
...
+    On a alors $\forall p \in \mathrm{P}$, $v_p(a) \leqslant v_p(b) \leqslant v_p(c)$ car $a$ divise $bc$.
+    Si $v_p(a) \geqslant b$, alors $v_p(b) = 0$.
+    Pour $H_p \in \mathrm{P}$, $v_p(a) \leqslant v_p(c)$, donc $a$ divise $c$, qui vérifie 3).
```
Comment (Yag000) [inline algebre-II-notes.tex]:
```suggestion
    Pour $H_p \in \mathcal{P}$, $v_p(a) \leqslant v_p(c)$, donc $a$ divise $c$, qui vérifie 3).
```

**Review by Yag000 — CHANGES_REQUESTED** [review]
Good, job, some minor modifications needed and it will be perfect for merge :)

**Comment by GabinDDL** [comment]
> Good, job, some minor modifications needed and it will be perfect for merge :)

**Comment by GabinDDL** [comment]
@Yag000 normally problems are solved

**Pull request closed without merging**